transformer_tools/mlflow_transformers_flavor.py

```diff
@@ -322,30 +341,87 @@ def load_model(model_uri, **kwargs):
     transformers_model_artifacts_path = os.path.join(
         local_model_path, transformers_conf["model_data"]
     )
-    return _load_model(path=transformers_model_artifacts_path, **kwargs)
+    return _load_model(path=transformers_model_artifacts_path, model_type=model_type, **kwargs)
 
 
-def mlflow_transformers_flavor(path="/mnt/models", **kwargs):
+def mlflow_transformers_flavor(
+    path="/mnt/models", model_type: ModelType = ModelType.SEQUENCE_CLASSIFICATION, **kwargs
+):
     """TODO: add docstring."""
     model_path = os.path.join(seldon_core.Storage.download(path))
-    return _load_pyfunc(model_path, **kwargs)
+    return _load_pyfunc(model_path, model_type=model_type, **kwargs)
 
 
-def _load_pyfunc(path, **kwargs):
+def _load_pyfunc(path, model_type: ModelType = ModelType.SEQUENCE_CLASSIFICATION, **kwargs):
     """Load PyFunc implementation.
 
     Called by ``pyfunc.load_pyfunc``.
 
     Args:
         path: Local filesystem path to the MLflow Model with the
             ``transformer_pretrained`` flavor.
+        model_type: Model type.
     """
-    return _TransformerPretrainedWrapper(
-        _load_model(path, **kwargs), _load_tokenizer(path, **kwargs)
-    )
+    model_wrapper: Any
```

Comment:
We could maybe narrow this type annotation to only the wrapper types that currently exist.

That is `Union[_TransformerSequenceClassificationWrapper, _TransformerSeq2SeqWrapper]`

But using a variable type annotation like this will require [Python 3.6+](https://www.python.org/dev/peps/pep-0526/#global-and-local-variable-annotations). Do we aim to support 3.5?

Reply:
> Do we aim to support 3.5?

No. At least 3.6+ for all of our projects. Some maybe 3.7+

Reply:
Changed. Thanks for the comment @twolffpiggott